app/models/player.rb

```diff
@@ -37,4 +37,8 @@ def defense_photo_url
     defense = Player.find_by(team: self.team, position: 'DEF')
     defense.photo_url if defense
   end
+
+  def current_team_player(team)
+    team_players.where(team_id: team.id).first
+  end
```

Comment:
I'm not sure I understand why the first team_player that matches the team_id should be the current_team_player.